GitHubCore.ps1

```diff
@@ -154,7 +151,9 @@ function Invoke-GHRestMethod
     # we'll just always continue the existing one...
     $stopwatch.Start()
 
-    $url = "$script:gitHubApiUrl/$UriFragment"
+    $gitHubApiUrl = "https://api.$(Get-GitHubConfiguration -Name "GitHubBaseUrl")"
```

Comment:
I'd propose a different breakdown for readability of components. (Can't use GitHub's suggestion feature since it only supports single-line):

```
$gitHubBaseUrl = Get-GitHubConfiguration -Name "GitHubBaseUrl"
$url = "https://api.$gitHubBaseUrl/$UriFragment"
```

or, you could do this:

```
$gitHubBaseUrl = Get-GitHubConfiguration -Name "GitHubBaseUrl"
$gitHubApiUrl = "https://api.$($gitHubBaseUrl)"
$url = "$gitHubApiUrl/$UriFragment"
```

While what you have already is totally valid, I think embedding function calls within a string definition reduces readability of the code.

Reply:
The readability concern is quite valid, I was going for minimal changes in my first iteration. I'll modify according to your recommendation.

Reply:
Reordered according to first suggestion, as I quite liked how the syntax highlighting looked and the statement reads from left to right.